src/eth_provider/provider.rs

```diff
@@ -5,11 +5,13 @@ use cainome::cairo_serde::CairoArrayLegacy;
 use eyre::Result;
 use itertools::Itertools;
 use mongodb::bson::doc;
+use reth_primitives::serde_helper::JsonStorageKey;
+use reth_primitives::serde_helper::U64HexOrNumber;
```

Comment:
Can we just group these in one import?

Reply:
Done. I think in the future we can use this

https://github.com/paradigmxyz/reth/blob/6801869c0d3e5ebf1b187d64694ec7ca13d809ea/rustfmt.toml#L2

in order to automatically classify the imports on save.